lib/schema-evolution-manager/args.rb

```diff
@@ -19,13 +19,14 @@ class Args
       }
 
       FLAGS_NO_ARGUMENTS = {
+        :pgpass => "Store the password in pgpass for the duration of the process",
         :dry_run => "Include flag to echo commands that will run without actually executing them",
         :help => "Display help",
         :verbose => "Enable verbose logging of all system calls",
       }
     end
 
-    attr_reader :artifact_name, :host, :port, :name, :prefix, :url, :user, :dir, :dry_run, :tag
+    attr_reader :artifact_name, :host, :port, :name, :prefix, :url, :user, :dir, :dry_run, :tag, :pgpass
```

Comment:
Since ".pgpass" file is automatically handled by the psql client should sem-apply simply fall back on  ".pgpass" being handled by the psql both to check the permissions (600 or more restrictive) and the correctness?